install_config/aggregate_logging.adoc

```diff
@@ -1518,3 +1518,627 @@ $ oc label node --all logging-infra-fluentd=true <1>
+
+| `pipeline_metadata.collector.ipaddr4`
+|IP address v4 of the collector server, can be an array.
+
+| `pipeline_metadata.collector.ipaddr6`
+|IP address v6 of the collector server, can be an array.
+
+| `pipeline_metadata.collector.inputname`
+|How the log message was received on the collector whether it was TCP/UDP, or
+imjournal/imfile.
+
+| `pipeline_metadata.collector.received_at`
+|Time when the message was received at the collector.
+
+| `pipeline_metadata.collector.original_raw_message`
+|The original non-parsed log message, collected by the collector or as close to the
+source as possible.
+|===
+
+[discrete]
+[[exported-fields-pipeline_metadata.normalizer]]
+=== `pipeline_metadata.normalizer` Fields
+
+This section contains metadata specific to the normalizer.
+
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
+| `pipeline_metadata.normalizer.hostname`
+|FQDN of the normalizer.
+
+| `pipeline_metadata.normalizer.name`
+|Name of the normalizer.
+
+| `pipeline_metadata.normalizer.version`
+|Version of the normalizer.
+
+| `pipeline_metadata.normalizer.ipaddr4`
+|IP address v4 of the normalizer server, can be an array.
+
+| `pipeline_metadata.normalizer.ipaddr6`
+|IP address v6 of the normalizer server, can be an array.
+
+| `pipeline_metadata.normalizer.inputname`
+|how the log message was received on the normalizer whether it was TCP/UDP.
+
+| `pipeline_metadata.normalizer.received_at`
+|Time when the message was received at the collector.
+
+| `pipeline_metadata.normalizer.original_raw_message`
+|The original non-parsed log message as it is received at the normalizer.
+
+| `pipeline_metadata.trace`
+|The field records the trace of the message. Each collector/normalizer appends
+information about itself and the date/time when the message was processed.
+|===
+
+[discrete]
+[[exported-fields-ovirt]]
+=== oVirt Fields
+
+Namespace for oVirt metadata.
+
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
+| `ovirt.entity`
+|The type of the data source, hosts, VMS, and engine.
+
+| `ovirt.host_id`
+|The oVirt host UUID.
+|===
+
+[discrete]
+[[exported-fields-ovirt.engine]]
+=== `ovirt.engine` Fields
+
+Namespace for oVirt engine related metadata. The FQDN of the oVirt engine is
+`ovirt.engine.fqdn`
+
+[discrete]
+[[exported-fields-aushape]]
+=== Aushape Fields
+
+Audit events converted with Aushape. For more information please see
+link:https://github.com/Scribery/aushape[Aushape].
+
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
+| `aushape.serial`
+|Audit event serial number.
+
+| `aushape.node`
+|Name of the host where the audit event occurred.
+
+| `aushape.error`
+|The error aushape encountered while converting the event.
+
+| `aushape.trimmed`
+|An array of JSONPath expressions relative to the event object, specifying
+objects/arrays with content removed as the result of event size
+limiting. Empty string means event itself. Empty array means trimming occurred
```

Comment:
Nick says:

Empty string means that the "event" object itself (i.e. the root) had
something removed from it.